ledger-api/grpc-definitions/com/daml/ledger/api/v1/version_service.proto

```diff
@@ -60,10 +60,14 @@ message FeaturesDescriptor {
 message UserManagementFeature {
   // Whether the Ledger API server provides the user management service.
   bool supported = 1;
+
   // The maximum number of rights that can be assigned to a single user.
-  // Value of 0 means that no rights per user limit is enforced.
+  // Servers MUST support at least 100 rights per user.
+  // A value of 0 means that the server enforces no rights per user limit.
   int32 max_rights_per_user = 2;
+
   // The maximum number of users the server can return in a single response (page).
-  // Value of 0 means that no page size limit is enforced.
+  // Servers MUST support at least a 100 users per page.
```

Comment:
This can be set from the CLI to any integer value, less then 100 or possibly even negative

Reply:
Thx for adding the checks to the CLI.

Can the CLI also be used to set the max_rights_per_user, or do we have a static value of 1k there?

Put differently: is the PR ready to merge from your view @pbatko-da ?

Reply:
It's a static value of 1k for max_rights_per_user.

Ok to merge